sdk/identity/Azure.Identity/src/Credentials/InteractiveBrowserCredentialOptions.cs

```diff
@@ -69,6 +69,6 @@ public string TenantId
         /// <summary>
         /// The options for customizing the browser for interactive authentication.
         /// </summary>
-        public BrowserCustomizationOptions BrowserCustomizedOptions { get; set; }
+        public BrowserCustomizationOptions BrowserCustomization { get; set; }
```

Comment:
Should this be `BrowserCustomizations`?

Reply:
I think the singular is what was intended and was the naming agreed upon, so I think I'll keep it this way.